docs/useThrottle.md

```diff
@@ -20,48 +20,72 @@ import { useThrottle } from 'rooks';
 
 ## Usage
 
-```jss
-function ThrottleDemo() {
-  const [number, setNumber] = useState(0);
-  const addNumber = () => setNumber(number + 1);
-  const [addNumberThrottled, isReady] = useThrottle(addNumber, 1000);
-  // isReady is a boolean that tells you whether calling addNumberThrottled at that point
+```jsx
+function Demo() {
+  const [text, setText] = useState("");
+  const [throttleValue, setThrottleValue] = useState("");
+  const [throttledFn, isReady] = useThrottle(changeThrottleValue, 1000);
+  // isReady is a boolean that tells you whether calling throttledFn at that point
   // will fire or not.
   // Once the timeout of 1000ms finishes, isReady will become true to indicate that the next time
-  // addNumberThrottled is called it will run right away.
+  // throttledFn is called it will run right away.
+  const ref = useRef(null);
+
+  function changeThrottleValue() {
+    setThrottleValue(ref.current.value);
+  }
+
   return (
-    <>
-      <h1>Number: {number}</h1>
-      <p>Click really fast.</p>
-      <button onClick={addNumber}>Add number</button>
-      <button onClick={addNumberThrottled}>Add number throttled</button>
-    </>
+    <div>
+      <h1>Rooks : useThrottle example</h1>
+      <input
+        ref={ref}
+        onChange={(e) => {
+          setText(e.target.value);
+          if (isReady) throttledFn();
```

Comment:
`isReady` is just an indication for the user. But the conditional is not needed here.

Reply:
Got it